### PR TITLE
updated haddock3 tutorials

### LINKED ISSUE #662: change clustering parameters' names in haddock3 tutorials
if PR https://github.com/haddocking/haddock3/pull/769 gets merged, some parameters' names must be changed also in the haddock3 tutorials

### PR DESCRIPTION
**Pull request by mgiulini**
Closes #662 by updating a bit the haddock3 tutorials

### CODE REVIEW

File: education/HADDOCK3/HADDOCK3-antibody-antigen/index.md [hunk 1]
```diff
@@ -629,11 +632,10 @@ binding site in the final models, while the residues defined as `active` (typica
 site residues) will. When using the HADDOCK server, `passive` residues will be automatically defined. Here since we are
 using a local version, we need to define those manually.
 
-This can easily be done using a script from our [haddock-tools][haddock-tools] repository, which is also provided for convenience
-in the `scripts` directly of the archive you downloaded for this tutorial:
+This can easily be done using a command line interface taken from our [haddock-tools][haddock-tools] repository:
```
Comment (amjjbonvin) [inline education/HADDOCK3/HADDOCK3-antibody-antigen/index.md:635]: I would rephrase to "This can easily be done using the following HADDOCK3 command line interface:"

File: education/HADDOCK3/HADDOCK3-antibody-antigen/index.md [hunk 2]
```diff
@@ -997,7 +991,7 @@ randremoval = false
 sampling = 10000
 
 [clustfcc]
-threshold = 10
+min_population = 10
```
Comment (amjjbonvin) [inline education/HADDOCK3/HADDOCK3-antibody-antigen/index.md:994]: I would change to `min_population = 4`

File: education/HADDOCK3/HADDOCK3-antibody-antigen/index.md [hunk 3]
```diff
@@ -997,7 +991,7 @@ randremoval = false
...
 
 [seletopclusts]
 ## select all the clusters
```
Comment (amjjbonvin) [inline education/HADDOCK3/HADDOCK3-antibody-antigen/index.md:997]: And add here: `top_models = 10` to select max 10 models per cluster

File: education/HADDOCK3/HADDOCK3-antibody-antigen/index.md [hunk 4]
```diff
@@ -1103,7 +1091,7 @@ unambig_fname = "restraints/antibody-unambig.tbl"
 randremoval = false
 
 [clustfcc]
-threshold = 10
+min_population = 10
```
Comment (amjjbonvin) [inline education/HADDOCK3/HADDOCK3-antibody-antigen/index.md:1094]: Change to min_population = 4

File: education/HADDOCK3/HADDOCK3-antibody-antigen/index.md [hunk 5]
```diff
@@ -1103,7 +1091,7 @@ unambig_fname = "restraints/antibody-unambig.tbl"
...
 
 [seletopclusts]
 ## select all the clusters
```
Comment (amjjbonvin) [inline education/HADDOCK3/HADDOCK3-antibody-antigen/index.md:1097]: Add as above `top_models = 10` to select max 10 models per cluster

File: education/HADDOCK3/HADDOCK3-antibody-antigen/index.md [hunk 6]
```diff
@@ -1201,7 +1183,7 @@ ambig_fname = "restraints/ambig-paratope-NMR-epitope-act.tbl"
 unambig_fname = "restraints/antibody-unambig.tbl"
 
 [clustfcc]
-threshold = 10
+min_population = 10
```
Comment (amjjbonvin) [inline education/HADDOCK3/HADDOCK3-antibody-antigen/index.md:1186]: set to 4 as above

File: education/HADDOCK3/HADDOCK3-antibody-antigen/index.md [hunk 7]
```diff
@@ -1201,7 +1183,7 @@ ambig_fname = "restraints/ambig-paratope-NMR-epitope-act.tbl"
...
 
 [seletopclusts]
 ## select all the clusters
```
Comment (amjjbonvin) [inline education/HADDOCK3/HADDOCK3-antibody-antigen/index.md:1189]: Select max 10 models per cluster by defining `top_models = 10`

**Comment by mgiulini** [comment]
Hi @amjjbonvin I implemented your (and Stefan's) suggestions, should be ready for the merge now.

**Review by amjjbonvin — APPROVED** [review]
